code/__HELPERS/unsorted.dm

```diff
@@ -1507,3 +1507,13 @@ proc/pick_closest_path(value, list/matches = get_fancy_list_of_atom_types())
 	else if(powerused < 1000000000) //Less than a GW
 		return "[round((powerused * 0.000001),0.001)] MW"
 	return "[round((powerused * 0.000000001),0.0001)] GW"
+
+// Bucket a value within boundary
+/proc/get_bucket(var/bucket_size, var/max, var/current, var/min = 0, var/list/boundary_terms = list())
```

Comment:
Feel free to remove the `var/` prefixes, they are already implied on arguments, also remove the new `list()` from `boundary_terms`, it's not used by default.